packages/date-picker/src/DatePicker/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { Dispatch, SetStateAction, useEffect, useRef } from "react"
+import DayPicker from "react-day-picker/DayPicker"
+import cx from "classnames"
+import { NavbarElementProps } from "react-day-picker/types/Props"
+import { Modifier, DayModifiers } from "react-day-picker/types/Modifiers"
+import { CalendarNav } from "../CalendarNav/CalendarNav"
+import { defaultCalendarClasses } from "./CalendarClasses"
+import calendarStyles from "./Calendar.scss"
+
+export type CalendarProps = {
+  setPopperElement: Dispatch<SetStateAction<HTMLDivElement | null>>
+  styles: { [key: string]: React.CSSProperties }
+  attributes: {
+    [key: string]:
+      | {
+          [key: string]: string
+        }
+      | undefined
+  }
+  classNameAndIHaveSpokenToDST?: string
+  value?: Date
+  initialMonth?: Date
+  firstDayOfWeek: number
+  disabledDays?: Modifier | Modifier[]
+  handleOnDayChange: (day: Date, modifiers: DayModifiers) => void
+  handleKeyDown: (e: any) => void
```

Comment:
Given these are props, I believe we agreed on `on` being the prefix instead of `handle`